### ISSUE TITLE
Spotlight search ↓/↑ to navigate right/left is confusing

### ISSUE THREAD

**Issue by HarHarLinks** [vector-im/element-web#20339]
### Your use case

![image](https://user-images.githubusercontent.com/2803622/147239442-4e24fe39-6d08-418e-be4e-c588b3b64173.png)

#### What would you like to do?

The recently viewed (breadcrumbs) are obviously arranged horizontally (left to right) while recent searches are listed vertically (top to bottom). After typing a search term, we're left with only a vertical list.

In the initial view, I would like to navigate recently viewed by using <kbd>←</kbd>/<kbd>→</kbd>. Pressing <kbd>↓</kbd>/<kbd>↑</kbd> should go straight into recent searches, as the layout suggests.

"Use <kbd>↓</kbd> <kbd>↑</kbd> to scroll results" should become "Use <kbd>↓</kbd> <kbd>↑</kbd> <kbd>←</kbd> <kbd>→</kbd> to navigate".

#### Why would you like to do it?

It is confusing to first traverse the list horizontally using the "vertical arrow keys" before actually moving vertically.
Similarly it's confusing that using "horizontal arrow keys" does not move at all, even though the list is horizontal.

### Have you considered any alternatives?

Arrange both lists in the same direction. I don't think that's a good idea, I would very much like the instant access to both lists that above interaction provides. You're also more likely to recognize recently viewed rooms by their avatar while for recent searches the name is very relevant (perhaps the main room alias should be shown as well)

**Comment by HarHarLinks** [comment]
@SimonBrandner A-Polls?

**Comment by SimonBrandner** [comment]
> @SimonBrandner A-Polls?

Sorry, I miss-typed something

**Comment by t3chguy** [comment]
> In the initial view, I would like to navigate recently viewed by using ←/→.

Your focus remains on a text field, left right must navigate the caret to maintain sane behaviour for screen reader users.

**Comment by HarHarLinks** [comment]
> > In the initial view, I would like to navigate recently viewed by using ←/→.
> 
> Your focus remains on a text field, left right must navigate the caret to maintain sane behaviour for screen reader users.

The horizontal "Recently viewed" list is only visible when the text field is empty and ←/→ have no effect. When you type text the "Recently viewed" disappear. Is it possible to implement this based on this mutual exclusion? The other possibility is to make recently viewed into a vertical list or (while no search term is entered) multiple vertical lists (switch with tab or so) instead.

Might not be possible, idk well enough how screen reader things work.

**Comment by t3chguy** [comment]
> The horizontal "Recently viewed" list is only visible when the text field is empty and ←/→ have no effect.

That is a great point, we should be able to make use of this yes

@niquewoodhouse not sure how to best show that to the user in the hint, that left-right only works on that specific horizontal list and not when you have something already typed in
